main.cpp

```diff
@@ -368,6 +406,19 @@ struct MusicStudio
     {
         std::cout << "MusicStudio Destructor" << std::endl;
     }
+
+    JUCE_LEAK_DETECTOR(MusicStudio)
+};
+
+struct MusicStudioWrapper
+{
+    MusicStudioWrapper(MusicStudio* MusicStudioPointer) : musicStudioptr(MusicStudioPointer){}
+    ~MusicStudioWrapper()
+    {
+        delete musicStudioptr;
+    }
+    
+    MusicStudio* musicStudioptr = nullptr;
```

Comment:
`camelCase` your member variable name.  
Type names begin with capital letters.  variables do not.
- [ ] revise